README.md

```diff
@@ -1,75 +1,90 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
 # Project Name
-
+Hello Microverse 
 > Description the project.
-
+The Project Displays hello Microverse 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages : HTML and CSS
+- Technologies used : Linters 
+
 
-## Live Demo (if available)
+## Getting Started
 
-[Live Demo Link](https://livedemo.com)
+To get a local copy up and running follow these simple example steps.
 
+### Prerequisites
+A code Editor, Git 
 
-## Getting Started
+### Install 
```

Comment:
Great job @AbhishekkM05 🤝

- [ ] Kindly restructure your readme file properly. It seems like you have two files in one file.
- [ ] Also remove the install line as it is not very clear when you click on its link.